Source/WebCore/rendering/RenderTheme.cpp

```diff
@@ -503,6 +503,9 @@ static RefPtr<ControlPart> createProgressBarPartForRenderer(const RenderObject&
 
 static RefPtr<ControlPart> createSliderTrackPartForRenderer(const RenderObject& renderer)
 {
+    if (!is<HTMLInputElement>(renderer.node()))
```

Comment:
You can use `dynamicDowncast` in line 513 to better organize the logic.

Reply:
Done.